backend/tracim_backend/config.py

```diff
@@ -1024,6 +1037,17 @@ def _check_global_config_validity(self) -> None:
                     )
                 ) from e
 
+        self.check_mandatory_param(
+            "RICH_TEXT_PREVIEW__TEMPLATE_PATH", self.RICH_TEXT_PREVIEW__TEMPLATE_PATH
+        )
+        self.check_file_path_param(
+            "RICH_TEXT_PREVIEW__TEMPLATE_PATH", self.RICH_TEXT_PREVIEW__TEMPLATE_PATH, readable=True
+        )
+        self.check_mandatory_param("RICH_TEXT_PREVIEW__CSS_PATH", self.RICH_TEXT_PREVIEW__CSS_PATH)
+        self.check_file_path_param(
+            "RICH_TEXT_PREVIEW__CSS_PATH", self.RICH_TEXT_PREVIEW__CSS_PATH, readable=True
+        )
+
```

Comment:
This make 2 new files from branding mandatory for all installations: https://github.com/tracim/tracim/issues/5711 I don't really like the fact we have such breaking change forcing sys-administrator to do manual fix but i don't see better solution for now, this feature doesn't make much sense to be optional and having tracim in a partially working state is not a good solution.

@PhilippeAccorsi What do you think about this kind of breaking change ?